integrations/apache/src/apache.go

```diff
@@ -1,19 +1,25 @@
 package main
 
 import (
+	"time"
+
 	sdk_args "github.com/newrelic/infra-integrations-sdk/args"
 	"github.com/newrelic/infra-integrations-sdk/log"
 	"github.com/newrelic/infra-integrations-sdk/sdk"
 )
 
 type argumentList struct {
 	sdk_args.DefaultArgumentList
-	StatusURL string `default:"http://127.0.0.1/server-status?auto" help:"Apache status-server URL."`
+	StatusURL    string `default:"http://127.0.0.1/server-status?auto" help:"Apache status-server URL."`
+	CABundleFile string `default:"" help:"Alternative Certificate Authority bundle file"`
```

Comment:
I'm not sure if this `default:""` it's needed. What happen if you omit `default`?

Reply:
Good catch. Just removed it and works.